### PR TITLE
Misp objects/#175

### PR DESCRIPTION
**Pull request by haxpak**
solves issue #175

### CODE REVIEW

File: objects/file/definition.json [hunk 1]
```diff
@@ -108,9 +108,9 @@
       "recommended": false
     },
     "malware-sample": {
-      "description": "The file itself (binary)",
-      "ui-priority": 1,
-      "misp-attribute": "malware-sample"
```
Comment (Rafiot) [inline objects/file/definition.json]: malware-sample is definitely correct here, and working: https://github.com/MISP/PyMISP/blob/master/tests/testlive_comprehensive.py#L995

How are you using the object?

Comment (haxpak) [reply objects/file/definition.json]: I was trying to import a text file.
if I place the file as a non malicious plain attachment, it does not render it as an object, which cannot be refrenced with other objects

Comment (Rafiot) [reply objects/file/definition.json]: Did you use the API, or the web interface?

**Comment by haxpak** [comment]
I am using the web interface

**Comment by Rafiot** [comment]
Oh, sorry, I misunderstood what you were doing. Right now, MISP always create a simple attribute, not object, for the sane attachments, and you can only create a file object for malwares. I strongly discourage you to change the template that way, because the malware-samples are compressed and encrypted, not the attachments.

@iglocska What do you think? Should we add an object reference "attachment" in the template and always create a file object when we add an attachment to an event?

**Pull request closed without merging**